pkg/standalone/standalone.go

```diff
@@ -191,7 +191,9 @@ func Init(runtimeVersion string, dockerNetwork string, redisHost string, slimMod
 	} else {
 		dockerContainerNames := []string{DaprPlacementContainerName, DaprRedisContainerName, DaprZipkinContainerName}
 		for _, container := range dockerContainerNames {
-			ok, err := confirmContainerIsRunningOrExists(utils.CreateContainerName(container, dockerNetwork), true)
+			// We don't need to check for the redis container if a custom host was specified
+			containerEnabled := container != DaprRedisContainerName || redisHost == daprDefaultHost
+			ok, err := confirmContainerIsRunningOrExists(utils.CreateContainerName(container, dockerNetwork), containerEnabled)
```

Comment:
why do we need to check against docker if the hostname specified is different from defaultHostName?

Reply:
This was following the logic found here:

https://github.com/dapr/cli/blob/master/pkg/standalone/standalone.go#L276

Rather than modify those methods to return more information which would touch more code, I just repeated the check.